svf-llvm/lib/LLVMModule.cpp

```diff
@@ -319,6 +320,17 @@ void LLVMModuleSet::initSVFBasicBlock(const Function* func)
             const SVFBasicBlock* svf_pred_bb = getSVFBasicBlock(*pred_it);
             svfbb->addPredBasicBlock(svf_pred_bb);
         }
+
+        // set entry / exit block
+        if (svfbb->getPredecessors().empty())
+        {
+            svfFun->setEntryBlock(svfbb);
+        }
+        if (svfbb->getSuccessors().empty())
+        {
+            svfFun->setExitBlock(svfbb);
```

Comment:
We should consider the unreachable blocks introduced by instructions such as exit(0). These blocks are not function exit blocks and cannot be connected to the unique exit block. A valid function exit should end with a return instruction.

Reply:
To my knowledge,  each llvm function can only have one return instruction. This means there should be only one function exit block? In that way, no extra unique function exit block is needed.

Reply:
Yeah, we do not need an extra unique exit block `theoretically`. 

I would like to improve robustness of SVF: when someone generate a `valid` bitcode file like above, we should handle it correctly.

And I have a question, why my first pr can't pass the testcases. I guess the following assert leads to fail to pass the testcase (`maybe multi exit basic block`)
```
inline void setExitBlock(SVFBasicBlock *bb)
{
    assert(!exitBlock && "function already has a exit Basicblock.");
    exitBlock = bb;
}
```

Because I have no testcases, so I can only guess the reason...